src/xrdreq/GetConfigQservRequest.cc

```diff
@@ -0,0 +1,93 @@
+/*
+ * LSST Data Management System
+ * Copyright 2018 LSST Corporation.
```

Comment:
Need to use the right header message.

Reply:
Good suggestion! Many headers are obsolete.